### PR TITLE
Use `MessageChannel` in `maker_routes`

### PR DESCRIPTION
**Pull request by luckysori**
More work towards #231.

### CODE REVIEW

File: daemon/src/routes_maker.rs [hunk 1]
```diff
@@ -144,55 +143,48 @@ pub struct PromptAuthentication {
 }
 
 #[rocket::post("/cfd/<id>/<action>")]
-pub async fn post_cfd_action(
+pub fn post_cfd_action(
     id: OrderId,
     action: CfdAction,
-    cfd_actor_address: &State<Address<maker_cfd::Actor>>,
+    user_choice_channel: &State<Box<dyn MessageChannel<maker_cfd::UserChoice>>>,
```
Comment (thomaseizinger) [inline daemon/src/routes_maker.rs]: Should we follow the other naming here and call this `maker_cfd::CfdAction`?

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/routes_maker.rs (hunk 1 above)
Comment (luckysori) [reply daemon/src/routes_maker.rs]: Are you referring to renaming `maker_cfd::UserChoice` to `maker_cfd::CfdAction`? I think `UserChoice` is clearer, but if we have a convention in place I'm happy to change it.

Comment (thomaseizinger) [reply daemon/src/routes_maker.rs]: The HTTP endpoint uses `CfdAction` for the exact same enum, minus the `OrderId` field. I'd suggest following the naming here for now and perhaps rename both at a later point.

**Pull request merged**